### PR TITLE
docs: share our issue management, planning, and release policies

### PR DESCRIPTION
**Pull request by EisenbergEffect**
# Pull Request

## 📖 Description

This PR shares with the community more about how we manage issues, plan our work, and schedule releases. About 80% of this PR represents practices that we are already adhering to. About 20% is a modification/adaptation of our existing practices to add slightly more rigor. I see this addition of a little more rigor as a transitional phase for our project that we'll probably stay in for a year or so before we evolve things a bit further around more detailed release milestone planning and timed major releases. One step at a time though.

## 👩‍💻 Reviewer Notes

Please read everything.

### General

- [ ] I have included a change request file using `$ yarn change`
- [ ] I have added tests for my changes.
- [ ] I have tested my changes.
- [x] I have updated the project documentation to reflect my changes.
- [x] I have read the [CONTRIBUTING](https://github.com/Microsoft/fast/blob/master/CONTRIBUTING.md) documentation and followed the [standards](https://www.fast.design/docs/community/code-of-conduct/#our-standards) for this project.

## ⏭ Next Steps

A follow up PR will add more documentation on the FAST steering committee and its role in the larger governance model of FAST.

### CODE REVIEW

File: sites/website/src/docs/community/engineering-process/issue-management.md
Comment (janechu) [inline sites/website/src/docs/community/engineering-process/issue-management.md:19]:
```suggestion
- Set proper expectations with the community regarding how issues are going to be handled
```

File: sites/website/src/docs/community/engineering-process/issue-management.md
Comment (janechu) [inline sites/website/src/docs/community/engineering-process/issue-management.md:23]: Who are the "feature teams"?

File: sites/website/src/docs/community/engineering-process/issue-management.md
Comment (nicholasrice) [inline sites/website/src/docs/community/engineering-process/issue-management.md:28]: You might consider linking to the specific policy for `status:needs-information` here. My first thought in reading this was "what constitutes a 'timely response'"

File: sites/website/src/docs/community/engineering-process/issue-management.md
Comment (nicholasrice) [inline sites/website/src/docs/community/engineering-process/issue-management.md:104]:
```suggestion
If the author does not post a response or updates the PR within **14 days**, the issue may be closed. If the author responds within **7 days** after the issue is closed, the issue will be re-opened. We recognize that you may not be able to respond immediately to our requests, we're happy to hear from you whenever you're able to provide the new information.
```
Above in the "Needs Information" section you use "may" instead of "will" when referencing closing an issue - using "will" struck me as a hard commitment we were making that maybe wasn't intentional?

Comment (EisenbergEffect) [reply sites/website/src/docs/community/engineering-process/issue-management.md:104]: I like the softer approach you recommend. In practice we probably won't do that exactly unless later we introduce automation for this. So, I agree "may" is better.

File: sites/website/src/docs/community/engineering-process/issue-management.md
Comment (EisenbergEffect) [reply sites/website/src/docs/community/engineering-process/issue-management.md:23]: I'd consider you and Bill a feature team. Nick and Wendy now as well. More broadly my team and Chris team. But I'm trying not to get into all the individual team break down and just generically referring to "feature teams" here.

File: sites/website/src/docs/community/engineering-process/release-planning.md
Comment (nicholasrice) [inline sites/website/src/docs/community/engineering-process/release-planning.md:57]: What are your thoughts on breaking this 'visual changes' portion into a section with a heading under 'Major Releases'? It is an important point, and it seems a little buried to me where it is now.

File: CONTRIBUTING.md
Comment (scomea) [inline CONTRIBUTING.md:190]: Do we cover "how an approved pr gets merged" anywhere, esp for contributors without elevated privs?

File: sites/website/src/docs/community/engineering-process/release-planning.md
Comment (EisenbergEffect) [reply sites/website/src/docs/community/engineering-process/release-planning.md:57]: yeah, that sounds good

File: CONTRIBUTING.md
Comment (EisenbergEffect) [reply CONTRIBUTING.md:190]: I think that is covered in the primary contributing doc.

File: CONTRIBUTING.md
Comment (awentzel) [inline CONTRIBUTING.md:186]: This label may also be great to pair with a GH Discussion...

File: sites/website/src/docs/community/engineering-process/issue-management.md
Comment (awentzel) [reply sites/website/src/docs/community/engineering-process/issue-management.md:28]: I consider a timely response to be less than the amount of time you would grow frustrated if you were blocked on an issue that you filed into some system.  For example, I file a lot of bugs and if they were not triaged in 24 hours for open source I would be disappointed.  So, IMHO I believe 12 hours is a timely response.  The faster we respond to our community the more they will praise us, and thus share their experience.  Especially, for first-timers, and newbies.

File: sites/website/src/docs/community/engineering-process/issue-management.md
Comment (awentzel) [inline sites/website/src/docs/community/engineering-process/issue-management.md:120]: Is there a link we can include for the .NET team?  Cross-linking with credible partners is great for SEO.

File: sites/website/src/docs/community/engineering-process/release-planning.md
Comment (awentzel) [inline sites/website/src/docs/community/engineering-process/release-planning.md:26]: Some headings capitalize all names and others do not.  For example, "Rough costing" and "Quarterly Planning".  Should we consistently name headings?

File: sites/website/src/docs/community/engineering-process/release-planning.md
Comment (awentzel) [inline sites/website/src/docs/community/engineering-process/release-planning.md:61]: I think you could remove the duplicate statement made above, and just keep this one.  Also, with a link to the .NET team.

**Review by awentzel — APPROVED** [review]

File: CONTRIBUTING.md
Comment (EisenbergEffect) [reply CONTRIBUTING.md:186]: I think that it might be better to keep the discussion directly in the Issue or PR so that it can be managed and discussed in one place. It might be difficult if one group of folks is discussing in the issue and then another group is in Discussions.

File: sites/website/src/docs/community/engineering-process/issue-management.md
Comment (EisenbergEffect) [reply sites/website/src/docs/community/engineering-process/issue-management.md:120]: I'll think I missed a few things. I'll check in and make sure this matches our other docs and is consistent throughout.

**Review by janechu — APPROVED** [review]

**Comment by EisenbergEffect** [comment]
@chrisdholt I'd like to get your review of this before merging 😄  I know you are very busy today. If you can spare some time tomorrow or Wednesday, then I can make updates and probably we can get it published this week.

**Review by chrisdholt — APPROVED** [review]

**Pull request merged**
